spirv_glsl.cpp

```diff
@@ -3146,9 +3146,32 @@ void CompilerGLSL::fixup_implicit_builtin_block_names(ExecutionModel model)
 			{
 				auto flags = get_buffer_block_flags(var.self);
 				if (flags.get(DecorationPerPrimitiveEXT))
+				{
 					set_name(var.self, "gl_MeshPrimitivesEXT");
+					set_name(type.self, "gl_MeshPerPrimitiveEXT");
+				}
 				else
+				{
 					set_name(var.self, "gl_MeshVerticesEXT");
+					set_name(type.self, "gl_MeshPerVertexEXT");
+				}
+			}
+		}
+
+		if (model == ExecutionModelMeshEXT && var.storage == StorageClassOutput && !block)
+		{
+			auto *m = ir.find_meta(var.self);
```

Comment:
having one check instead of 3 feel more on cosmetic side. I've decided to pick current style of writing, to have more flatten code.
Can change to `if (m) { if (builtin_type == BuiltInPrimitivePointIndicesEXT) {} ... } `, if this is important